ramp-database/rampdb/model/submission.py

```diff
@@ -56,16 +56,101 @@
 
 
 class Submission(Model):
-    """An abstract (untrained) submission."""
-
+    """Submission table.
+
+    Parameters
+    ----------
+    name : str
+        The submission name.
+    event_name : str
```

Comment:
the init uses `event_team` ? And maybe it's not a string but an actual EventTeam object?